Fixed heroes.json

j1gs4w opened the pull request:
Changed Lycan, Wisp and Arc Warden to match
official Dota 2 names and your name conventions

Cst2989 commented:
This is awesome. 

The only problem is I keep geting an erorr if the id doesnt have a cooldown, it should return 0 or something.

Also, it would be super if the ability had a is_ultimate true or false for each ability if you can.

Thanks a lot.

j1gs4w commented:
Sorry i changed the order of checks in the method and messed it up by that.^^
I added a method isUltimate which checks if an ability has 3 levels or is invokers Ultimate because it has 4 levels, hope it works like it should.
I'll add a boolean to the abilities.json later for that when i'm back on my computer again.
